interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is a tool that allows you to use javascript and html elements in the same code. To use javascript functionality in JSX, you need to wrap it in curly braces.
 
-Researched answer:
+Researched answer: JSX is a syntax extension of Javascript designed to build components in React. It allows a combination of HTML tags and javascript functionality.
 
 2. What is yarn? What file(s) are modified in a React application when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: yarn is a tool used from the terminal to install and run React apps. It can install the various modules, .lock and .json file associated with React.
 
-Researched answer:
+Researched answer: Yarn is a package manager for React that fetches and downloads dependencies for creating a React app. 
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: An anonymous function is an unnamed function, as in it is not directly attached to a variable. I believe, for example, the functions inside higher order functions would be considered anonymous.
 
-Researched answer:
+Researched answer: Anonymous functions are not named, and cannot be accessed again after their creation unless invoked through a parent function.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: State is data saved locally through user input, props are bits of data made available across React components through props syntax.
 
-Researched answer:
+Researched answer: State is data saved locally TO IT'S COMPONENT only, whereas props pass data between components.
 
 5. What is the DOM?
 
-Your answer:
+Your answer: DOM stands for document object model. I'm not yet sure on the technicalities of it, but I believe it represents the UI as a user would while interacting with it. 
 
-Researched answer:
+Researched answer: The DOM is a representation of a created web page that can be displayed in the browser during development.
 
 6. STRETCH: Which is the difference between a div and a span?
 
-Your answer:
+Your answer: I know a div is used to "lump" together elements in html and jsx as a single customizable element. I'm not sure when it comes to span, but will research it and can get a better answer back to you.
 
-Researched answer:
+Researched answer: They are similar in their use for grouping elements together for common styling and use, but span is an inline element.
```

Comment:
Great work in this section!  Some of these answers could be expanded a little further, but overall you did a good job here.